docs/usecases/running_tests_on_cluster_not_deployed_by_ocsci.rst

```diff
@@ -0,0 +1,218 @@
+
+Access to cloud object storage (MCG only)
+-----------------------------------------
+
+If you are going to run some of `MCG test cases`_, you also need to create
+``data/auth.yaml`` file inside of ocs-ci directory, with credentials for cloud
+object storage accounts used by MCG.
+
+Minimal structure of the ``auth.yaml`` file is currently described in `this
+github comment
+<https://github.com/red-hat-storage/ocs-ci/issues/2649#issuecomment-671037451>`_.
+
+.. note::
+
+    Known issues related to ``auth.yaml`` file:
+
+    - `#2649`_ Document changes to auth.yaml to be used by CloudManager
+    - `#2623`_ Pick a source of truth for auth.yaml
+
+.. _`MCG test cases`: https://github.com/red-hat-storage/ocs-ci/tree/master/tests/manage/mcg
+.. _`#2649`: https://github.com/red-hat-storage/ocs-ci/issues/2649
+.. _`#2623`: https://github.com/red-hat-storage/ocs-ci/issues/2623
+
+
+What if I need to run tests on a new platform?
+----------------------------------------------
+
+Then you need to either try to reuse config for a similar platform, but whether
+such option is valid depends on a nature of a test cases to be executed and the
+platform itself.
+
+Another option is to create a minimal patch for ocs-ci which defines the new
+platform enough to allow a test execution. See example of `such patch for Azure
+platform <https://github.com/red-hat-storage/ocs-ci/pull/2056/files>`_. Again,
+such quick approach is not sufficient for some platform specific test cases.
+
+What if I see lot of warnings about catalogsource?
+--------------------------------------------------
+
+If you are using stable (aka live or released) version of OCS and specified OCS
+version via ``run-ci`` options (including ``--live-deploy`` option) as
+explained above, warnings similar to the ones show below the are a red
+herring::
+
+    Failedtogetresource:ocs-catalogsourceofkind:CatalogSource,selector:None,Error:Errorduringexecutionofcommand:oc-nopenshift-marketplace--kubeconfig/home/ocsqe/data/mbukatov-dc2-cluster_20200226T131741/auth/kubeconfiggetCatalogSourceocs-catalogsource-n
+    openshift-marketplace-oyaml.
+    ErrorisErrorfromserver(NotFound):catalogsources.operators.coreos.com"ocs-catalogsource"notfound
+
+    Numberofattemptstogetresourcereached!
+    Failedtogetresource:ocs-operatorsourceofkind:OperatorSource,selector:None,Error:Errorduringexecutionofcommand:oc-nopenshift-marketplace--kubeconfig/home/ocsqe/data/mbukatov-dc2-cluster_20200226T131741/auth/kubeconfiggetOperatorSourceocs-operatorsourc
+    e-nopenshift-marketplace-oyaml.
+    ErrorisErrorfromserver(NotFound):operatorsources.operators.coreos.com"ocs-operatorsource"notfound
+
+    Numberofattemptstogetresourcereached!
+
+And as such can be safely ignored. This is a known ocs-ci issue `#1556`_.
+
+So if you are running into some problem and see this error, you need to inspect
+the logs further to find the real problem.
+
+.. _`#1556`: https://github.com/red-hat-storage/ocs-ci/issues/1556
```

Comment:
I would also add a small section about cluster teardown

Reply:
The text is discussing test execution only, neither deployment nor teardown is involved, as noted above.

Reply:
Addressed in b60e5638bc45e7b9939287c90366f949ec8c2483